From b911a9862f04655f2119d8854946d1b94000c074 Mon Sep 17 00:00:00 2001
From: cnauroth <cnauroth@apache.org>
Date: Thu, 23 Apr 2015 13:10:39 -0700
Subject: [PATCH 1708/2848] HADOOP-11872. "hadoop dfs" command prints message
 about using "yarn jar" on Windows(branch-2 only).
 Contributed by Varun Vasudev.

(cherry picked from commit d1253f695df67ff27963d8ea4734b8a9aa79565b)
(cherry picked from commit 37673268efa022c5ec65ed7d88d60d796e965429)

Change-Id: Ie34fc740285d6708cd9d2203217286c039bab69e
---
 .../hadoop-common/src/main/bin/hadoop.cmd          |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-common-project/hadoop-common/src/main/bin/hadoop.cmd b/hadoop-common-project/hadoop-common/src/main/bin/hadoop.cmd
index f0693c8..2619eca 100644
--- a/hadoop-common-project/hadoop-common/src/main/bin/hadoop.cmd
+++ b/hadoop-common-project/hadoop-common/src/main/bin/hadoop.cmd
@@ -241,6 +241,8 @@ call :updatepath %HADOOP_BIN_PATH%
   @echo   fs                   run a generic filesystem user client
   @echo   version              print the version
   @echo   jar ^<jar^>            run a jar file
+  @echo                        note: please use "yarn jar" to launch
+  @echo                              YARN applications, not this command.
   @echo   checknative [-a^|-h]  check native hadoop and compression libraries availability
   @echo   distcp ^<srcurl^> ^<desturl^> copy file or directories recursively
   @echo   archive -archiveName NAME -p ^<parent path^> ^<src^>* ^<dest^> create a hadoop archive
-- 
1.7.9.5

